### PR TITLE
Check user fund conversion and create informational emails accordingly

### PR DESCRIPTION
**Pull request by aleksandraisabel**
Checked user's conversion data and refined email email event listener so it sends notices of options about fund transferring.

Email text yet in need of improving.

ConversionResponse class test coverage only 42% -> potential need for testing Conversion class

### CODE REVIEW

File: src/test/groovy/ee/tuleva/onboarding/mandate/email/MandateEmailServiceSpec.groovy [hunk 1]
```diff
@@ -25,23 +27,34 @@ class MandateEmailServiceSpec extends Specification {
 
     def "Send second pillar mandate email"() {
         given:
-        emailContentService.getSecondPillarHtml(Locale.ENGLISH) >> "html"
+        def user = sampleUser().build()
+        def isFullyConverted = false
+        def isThirdPillarActive = false
+        def conversion = ConversionResponseFixture.notFullyConverted()
```
Comment (ErkoRisthein) [inline src/test/groovy/ee/tuleva/onboarding/mandate/email/MandateEmailServiceSpec.groovy:33]: Static import `ConversionResponseFixture`

File: src/test/groovy/ee/tuleva/onboarding/mandate/listener/MandateEmailSenderSpec.groovy [hunk 2]
```diff
@@ -25,32 +25,47 @@ class MandateEmailSenderSpec extends Specification {
...
     MandateEmailService mandateEmailService = Mock(MandateEmailService)
 
+    @SpringBean
+    UserConversionService userConversionService = Mock(UserConversionService)
+
     def "send email when second pillar mandate event was received" () {
         given:
         User user = sampleUser().build()
+
+        UserPreferences contract = new UserPreferences()
```
Comment (ErkoRisthein) [inline src/test/groovy/ee/tuleva/onboarding/mandate/listener/MandateEmailSenderSpec.groovy:35]: contract?

File: src/test/groovy/ee/tuleva/onboarding/mandate/listener/MandateEmailSenderSpec.groovy [hunk 3]
```diff
@@ -25,32 +25,47 @@ class MandateEmailSenderSpec extends Specification {
...
+
         SecondPillarMandateCreatedEvent event = new SecondPillarMandateCreatedEvent(
             user, 123, "123".bytes, Locale.ENGLISH
         )
+        1 * episService.getContactDetails(_) >> contract
+
+        def conversion = ConversionResponseFixture.notFullyConverted()
```
Comment (ErkoRisthein) [inline src/test/groovy/ee/tuleva/onboarding/mandate/listener/MandateEmailSenderSpec.groovy:42]: Static import `ConversionResponseFixture`

File: src/test/groovy/ee/tuleva/onboarding/mandate/listener/MandateEmailSenderSpec.groovy (hunk 2 above)
Comment (aleksandraisabel) [reply src/test/groovy/ee/tuleva/onboarding/mandate/listener/MandateEmailSenderSpec.groovy]: "contract" taken from existing example. Changed to "userPreferences" throughout file

**Pull request merged**